galaxy_ng/app/api/ui/viewsets/user.py

```diff
@@ -0,0 +1,49 @@
+from django.shortcuts import get_object_or_404
+
+from rest_framework import mixins
+from rest_framework import permissions as drf_permissions
+
+from galaxy_ng.app.models import auth as auth_models
+from galaxy_ng.app.api import permissions
+from galaxy_ng.app.api.ui import serializers
+from galaxy_ng.app.api import base as api_base
+
+
+class UserViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
+    api_base.GenericViewSet,
+):
+
+    def get_serializer_class(self):
```

Comment:
minor: Because it just returns static class, it can be defined as `serializer_class = ` class attribute.